profiles/models.py

```diff
@@ -1,3 +1,42 @@
+import os
+
+from django.contrib.auth.models import User
+from django.core.validators import RegexValidator
 from django.db import models
+from django.db.models.signals import post_save
+
+
+# Django provides a table called user that stores basic user information like username, password and email id.
+class Pcuser(models.Model):
```

Comment:
Instead of creating extra model which refers to User Model we can add fields to Django User Model by inheriting the AbstractUser to the model. Also, it will also eliminate the need to use signals to create Pcuser instance whenever User is created.